timescaledb/tutorials/nfl-fantasy-league.md

```diff
@@ -18,7 +18,7 @@ get started with TimescaleDB and explore a real world dataset with SQL and Pytho
 * Python 3
 * TimescaleDB (see [installation options][install-timescale]) 
 * [Psql][psql-install] or any other PostgreSQL client (e.g. DBeaver)
-* Pandas and Matplotlib (`pip install pandas matplotlib`)
+* Pandas, Matplotlib, and Numpy (`pip install pandas matplotlib numpy`)
```

Comment:
I'm not 100% on this, but isn't numpy already included if you install pandas?

Reply:
haha your totally right! I can remove this real quick